lib/Models/GlobeOrMap.js

```diff
@@ -91,4 +91,14 @@ GlobeOrMap.prototype._createEntityFromImageryLayerFeature = function(feature) {
     return entity;
 };
 
+//set text attribution
+
+GlobeOrMap.prototype.addAttributionText = function(textAttribution){
+    return textAttribution;
```

Comment:
This should throw a `DeveloperError` as a reminder to implement it in the derived class.  See the other methods in this file, such as `captureScreenshot`.  Same below.